pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -265,13 +276,15 @@ func (reconciler *ReconcileKieApp) GetClient() client.Client {
 
 // Check ImageStream
 func (reconciler *ReconcileKieApp) checkImageStreamTag(name, namespace string) bool {
+	log := log.WithValues("kind", "ImageStreamTag", "name", name, "namespace", namespace)
 	result := strings.Split(name, ":")
 	if len(result) == 1 {
 		result = append(result, "latest")
 	}
 	tagName := fmt.Sprintf("%s:%s", result[0], result[1])
 	_, err := reconciler.imageClient.ImageStreamTags(namespace).Get(tagName, metav1.GetOptions{})
 	if err != nil {
+		log.V(1).Info("Object does not exist")
```

Comment:
Is 1 for info? I would think error would be 0, warning would be 1, and info would be 2 or higher? This new logger, btw, leaves much to be desired. I'm thinking maybe we should provide our own layer of indirection.

Reply:
So are error and warning considered implicitly known levels and then it starts counting up from 0 for Info? Do you have a link for that?

As far as adding a layer, I'm not suggesting we stop using this logger as you have. I'm suggesting having Error(), Warn(), Info(), and Debug() functions in one Go class and in there, call log.Error or log.V(1) and so on, as appropriate. The benefit would be that if we decide to make a chance, it's just in one place.

Reply:
We use the following -
https://github.com/kubernetes-sigs/controller-runtime/blob/v0.1.8/pkg/runtime/log/log.go#L43-L68

`logr` is what is leveraged from the controller-runtime log pkg -
https://github.com/go-logr/logr#differences-from-daves-ideas

Reply:
only debug, error, and info are currently recognized levels... stops at 1.

Reply:
Got it RE levels... my point stands about the extra layer. Not sure if you meant to respond to that as well.

Reply:
did we want to do something with this PR or separately?

Reply:
Let's go this PR, because it's the exact same lines being changed here.